### DISCUSSION TITLE
What would the official compatible Laravel versions for the Pest be?

### DISCUSSION THREAD

**Discussion by HassanZahirnia** [pestphp/pest#759] [Q&A] [answered]
Hey! 👋😊
I've added Pest to my newly launched website: https://laravel-package-ocean.com

(Feedbacks are welcome)

There is a small detail missing though, I'm not sure about the compatible Laravel versions? If you could tell me that I think that would be a great small detail to add.

Thanks for your hard work on Pest! 🌸🚀

**Comment by fabio-ivona** [comment, marked as answer]
Pest is compatible with PHPUnit 10, so only Laravel 10+ is supported

**Reply by HassanZahirnia** [reply]
Thanks a lot!

**Reply by fabio-ivona** [reply]
sorry, to be clear: Pest v2 is compatible with Laravel 10+

Pest v1 is compatible up to Laravel 9

**Reply by HassanZahirnia** [reply]
No problem! I will just mark it as 9+ compatible then.
